components/docker-auth/cmd/authn/authentication.go

```diff
@@ -0,0 +1,153 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/dgrijalva/jwt-go"
+	"github.com/golang/glog"
+
+	"github.com/cellery-io/cellery-hub/components/docker-auth/pkg"
+)
+
+var (
+	idpKey        []byte
+	dockerAuthKey []byte
+	err           error
+)
+
+const (
+	issuerClaim = "iss"
+	subjectClaim = "sub"
+	expirationClaim = "exp"
+	idpCertEnvVar = "IDP_CERT"
+	authTokenIssuer = "REGISTRY_AUTH_TOKEN_ISSUER"
+	dockerAuthCertEnvVar = "REGISTRY_AUTH_TOKEN_ROOTCERTBUNDLE"
+)
+
+func errLog(err error) {
```

Comment:
Why is this method required ? Let's remove this method and use glog along with a os.Exit to terminate.